rts/lra/lra-coordinator-jar/README.md

```diff
@@ -0,0 +1,30 @@
+Enable LRA JWT Security
+=======================
+
+
+Requirements
+------------
+
+* Always name the JWT token **Realm** as: `narayana-lra-jwt`
```

Comment:
I would like to understand where is this `realm` configured. Is it something automatic defined how the Smallrye JWT works?

Reply:
Actually, the `realm` will be configured as part of authentication/authorization mechanism like keycloak.